src/overlays/actors/ovl_En_Zl1/z_en_zl1.c

```diff
@@ -15,51 +15,650 @@ void EnZl1_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnZl1_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnZl1_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+void func_80B4AE18(EnZl1* this);
+void func_80B4AF18(Actor* thisx, GlobalContext* globalCtx);
+void func_80B4B010(Actor* thisx, GlobalContext* globalCtx);
+void func_80B4B240(Actor* thisx, GlobalContext* globalCtx);
+void func_80B4B8B4(Actor* thisx, GlobalContext* globalCtx);
+void func_80B4BBC4(Actor* thisx, GlobalContext* globalCtx);
+void func_80B4BC78(Actor* thisx, GlobalContext* globalCtx);
+void func_80B4BF2C(Actor* thisx, GlobalContext* globalCtx);
+
+#include "z_en_zl1_cutscene_data.c"
+
 const ActorInit En_Zl1_InitVars = {
-    ACTOR_EN_ZL1,
-    ACTORTYPE_NPC,
-    FLAGS,
-    OBJECT_ZL1,
-    sizeof(EnZl1),
-    (ActorFunc)EnZl1_Init,
-    (ActorFunc)EnZl1_Destroy,
-    (ActorFunc)EnZl1_Update,
-    (ActorFunc)EnZl1_Draw,
+    ACTOR_EN_ZL1, ACTORTYPE_NPC, FLAGS, OBJECT_ZL1, sizeof(EnZl1), EnZl1_Init, EnZl1_Destroy, EnZl1_Update, EnZl1_Draw,
```

Comment:
we've been leaving the casts to ActorFunc, which will also prevent clang from ruining the format